src/stateless/slider.vue

```diff
@@ -0,0 +1,597 @@
+<template>
+    <div class="slider-container" :class="Object.assign(sizeClass(), themeClass())">
+
+        <div class="slider-header">
+            {{ label }}
+        </div>
+
+        <div class="slider-unit">
+
+            <div class="slider-input">
+                <input-element
+                    :type="inputType"
+                    :size="size"
+                    :disabled="disabled"
+                    :value="currentValue"
+                    :step="step"
+                    :min-value="0"
+                    :max-value="999"
+                    :is-valid="isValidInput"
+                    :theme="theme"
+                    :alignment="alignment"
+                    @input="handleInput"
+                ><span slot="right">%</span></input-element>
+            </div>
+
+            <div class="slider-wrapper" :class="stateClass()"  @mousedown="startDrag" @keydown="keyboard" ref="slider" tabindex="0">
+                <div class="slider-content">
+                    <div class="slider-label">
+
+                        <div class="slider-label__unit" :class="labelClass(0)" ref="min">{{ minLabelValue }}</div>
```

Comment:
It doesn't make much sense for `slider-label` to be block name. This should probably be `slider__label-unit`.